### DISCUSSION TITLE
how to enable error code in diagnostics?

### DISCUSSION THREAD

**Discussion by semi710** [nvimdev/lspsaga.nvim#1062] [Q&A] [answered]
In diagnostics how can i enable this 
![image](https://github.com/nvimdev/lspsaga.nvim/assets/60490474/72e6e9dd-a991-4d81-b718-1ecd75983774)
error code?

**Comment by glepnir** [comment]
if has code it will show in diagnostic jump window you mean in show window ?

**Reply by semi710** [reply]
![image](https://github.com/nvimdev/lspsaga.nvim/assets/60490474/84bf86a5-d436-4b73-b667-4af9d6942e0a)
This is lsp saga line diagnostics

![image](https://github.com/nvimdev/lspsaga.nvim/assets/60490474/702ce2fc-d9ef-440e-a4f0-35f65989fbef)
and this is the same with the fn
```lua
function()
          local float = vim.diagnostic.config().float

          if float then
            local config = type(float) == "table" and float or {}
            config.scope = "line"
            config.border = "single"
            vim.diagnostic.open_float(config)
          end
        end,

```

**Reply by semi710** [reply]
i just want lspsaga diagnostic to also add the error code in the end
as sometimes it becomes easier to get to the root of the cause by the code.

**Comment by glepnir** [comment, marked as answer]
yep . in show window it not added but there already have an issue for it I remember . and it will support in 0.3 I already changed it in next verison
